lib/bundles/inspec-compliance/api.rb

```diff
@@ -5,13 +5,16 @@
 require 'net/http'
 require 'uri'
 
+require_relative 'api/login'
```

Comment:
should we move this file to the api directory too? Feels strange to have a `api.rb` and `api` directory. What do you think?

Reply:
That's actually pretty standard in Ruby.

Reply:
Yeah, it's common in Ruby. I'm fine with it if you are @chris-rock.